drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -127,6 +129,13 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
         default="",
     )
 
+    max_items_per_zone = Integer(
+        display_name=_("Maximum items per zone"),
+        help=_("This setting limits the number of items that can be dropped into a single zone"),
```

Comment:
@haikuginger Please add a period to make this consistent with help messages for other fields, and update the translation files.